tests/data/samplesheets/samplesheet-hamming.csv

```diff
@@ -0,0 +1,4 @@
+sample,mlst_alleles,metadata_1,metadata_2,metadata_3,metadata_4,metadata_5,metadata_6,metadata_7,metadata_8
+sample1,https://raw.githubusercontent.com/phac-nml/gasclustering/fix/distances/tests/data/reports/case-hamming/sample1.mlst.subtyping.json,,,,,,,,
+sample2,https://raw.githubusercontent.com/phac-nml/gasclustering/fix/distances/tests/data/reports/case-hamming/sample2.mlst.subtyping.json,,,,,,,,
+sample3,https://raw.githubusercontent.com/phac-nml/gasclustering/fix/distances/tests/data/reports/case-hamming/sample3.mlst.subtyping.json,,,,,,,,
```

Comment:
I think these file paths (`fix/distances`) may disappear when it's merged into dev and the branch deleted. You may want to either immediately change them after merging into dev, or preemptively push the test data to dev and update these to point there.